c10/core/DispatchKeySet.h

```diff
@@ -294,7 +294,10 @@ class DispatchKeySet final {
                              DispatchKey::Dense,
                              DispatchKey::Quantized,
                              DispatchKey::Sparse,
+                             DispatchKey::SparseCsr,
+                             DispatchKey::NestedTensor,
```

Comment:
NestedTensor?

Reply:
Yeah, `NestedTensor` was made to be per-backend a few days ago, and it missed a spot here :(. This one in particular is pretty hard to catch with tests - `has_any(DispatchKeySet)` is really an optimization over calling `has()` multiple times, where it's stated above that you should know what you're doing first - in particular, the keyset that you pass in isn't allowed to have *both* per-backend-functionality bits, and backend bits. This is a debug-only assert that you passed in a valid argument.

I'll add a more explicit comment in `DispatchKey.h` telling you to look for all of the code snippets with `[Note: Per-Backend Functionality Dispatch Keys]` to know where to add new per-backend keys.